### DISCUSSION TITLE
Display Program overview Info

### DISCUSSION THREAD

**Discussion by AdEmergency625** [astashov/liftosaur#176] [Ideas]
It would be good for the ability to get program or workout overview info. 

For example, I would like to set up all my equipment, recap program goals, and mentally prepare before I start my workout. So I would like to scan through those notes before my workout. 

Another example, add the max record from last time I use the program which could be months ago. So I could easily compare and evaluate my progression more than simple graphs available.

**Comment by 0xblur** [comment]
There is a program overview feature on the web editor that allows you to print your program and program stats.
